.github/workflows/release-automation.yml

```
@@ -0,0 +1,362 @@
+        - minor
+        - major
+      prerelease:
+        description: 'Is this a pre-release?'
+        type: boolean
+        default: false
+
+jobs:
+  changelog-and-release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)